draw/src/main/java/org/odk/collect/draw/QuitDrawingDialog.kt

```diff
@@ -0,0 +1,50 @@
+package org.odk.collect.draw
+
+import android.app.Activity
+import androidx.appcompat.app.AlertDialog
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import org.odk.collect.draw.databinding.QuitDrawingDialogLayoutBinding
+
+object QuitDrawingDialog {
+
+    @JvmStatic
+    fun show(
+        activity: Activity,
+        onDiscardChanges: Runnable?,
+        onSaveChanges: Runnable?
+    ): AlertDialog {
+        return create(
+            activity,
+            onDiscardChanges,
+            onSaveChanges
+        ).also {
+            it.show()
+        }
+    }
+
+    private fun create(
+        activity: Activity,
+        onDiscardChanges: Runnable?,
+        onSaveChanges: Runnable?
+    ): AlertDialog {
+        val binding = QuitDrawingDialogLayoutBinding.inflate(activity.layoutInflater)
+        val dialog = MaterialAlertDialogBuilder(activity)
+            .setTitle(org.odk.collect.strings.R.string.save_drawing_title)
+            .setView(binding.root)
+            .create()
+
+        binding.discardChanges.setOnClickListener {
```

Comment:
I can't see any test coverage for these 3 buttons - I'm guessing that's an existing problem though. This is probably a good opportunity to add it (maybe as part of a `DrawActivityTest`).